ReactPeerProgram/hpcard/src/Theme/theme.ts

```diff
@@ -0,0 +1,36 @@
+import { createTheme } from "@mui/material";
+import "@fontsource/montserrat";
+
+const theme = createTheme({
+  palette: {
```

Comment:
check the colors in the figma you can see text/high emphasis that should be included in the palette:{text:{high:color,low:color,medium:color}} check theme in material ui website